portfolio/src/main/webapp/index.jsp

```diff
@@ -128,32 +128,45 @@ <h1>Work Experience</h1>
               
               <dt>Foundant Technologies</dt>
               <dt class="workposition">Spring 2020 - Client Services Intern</dt>
-              <dd>Assist clients that use Foundant’s grant and scholarship management software or fund accounting software with any technical support questions or issues they might have.</dd>
+              <dd>Assist clients that use Foundant's grant and scholarship management software or fund accounting software with any technical support questions or issues they might have.</dd>
               
               <dt>Silver Lining Entertainment</dt>
               <dt class="workposition">October 2016 to November 2018 - Talent Manager's Assistant</dt>
               <dd>Managed desk of 20+ actors and coordinated between production, publicists, casting, and clients. Logged over 2,000 company commissions and increased efficiency by digitizing the tracking system.</dd>
               
               <dt>USC Annenberg - Media, Diversity, & Social Change Initiative</dt>
               <dt class="workposition">Summer 2015 - Student Research Assistant</dt>
-              <dd>Analyzed and quantified TV & Film content metrics for the 2014-2015 study: ‘Inclusion or Invisibility? The Comprehensive Annenberg Report on Diversity in Entertainment’ as seen <a href="https://bit.ly/2oBKDUr">here.</a></dd>
+              <dd>Analyzed and quantified TV & Film content metrics for the 2014-2015 study: 'Inclusion or Invisibility? The Comprehensive Annenberg Report on Diversity in Entertainment' as seen <a href="https://bit.ly/2oBKDUr">here.</a></dd>
       </section>
 
       <!-- Contact Me Section -->
-      <section id="contactme">
+
+      <%@ page import = "com.google.appengine.api.users.UserService, com.google.appengine.api.users.UserServiceFactory" %>
+      <%
+          UserService userService = UserServiceFactory.getUserService();
+          String userId = "";
+          if (userService.isUserLoggedIn()) {
+            userId = userService.getCurrentUser().getUserId();
+          }
+      %>
+      <section id="contactme" data-user-id="<%=userId%>">
           <h1>Contact Me</h1>
           <p>Please feel free to reach out to me through any of the below platforms or leave a comment below:</p>
 
+          <%
+          if (userService.isUserLoggedIn()) {
+            String urlToRedirectToAfterUserLogsOut = "/";
+            String logoutUrl = userService.createLogoutURL(urlToRedirectToAfterUserLogsOut);
+
+          %>
+          <p class="login-messages">(You are logged in as <%=userService.getCurrentUser().getEmail()%>, to logout <a href="<%=logoutUrl%>">click here</a>)</p>
```

Comment:
"click here" isn't generally what you want in the text of a hyperlink, since, e.g., screen readers end up saying "click here" to their users without much context when they tab through the page looking for links. Rather, make the link text descriptive and optionally trust that users know they can click on it, as you did for the log-in link.

Reply:
Got it! To clarify, would it be best to rephrase also the "Click here to login" to just "Login" (where the whole word is the link)?

Reply:
Either sounds fine to me.